src/Hl7.Fhir.Specification.Tests/Hl7.Fhir.Specification.Tests.csproj

```diff
@@ -47,6 +47,7 @@
   <ItemGroup>
```

Comment:
This ItemGroup is unnecessary, because it is already covered by the previous ItemGroup (with wildcards). So I would remove the whol ItemGroup here.

Reply:
Ah, yes, I've seen VS do that before. I remove them from time to time manually...